Month1/Week2/Section1/Exercise3/Main3.cpp

```diff
@@ -73,18 +78,21 @@ class FCircle
 	float DiameterCircle;
 public:
 	FCircle()
+	{
+	}
+	FCircle(float RadiusCircle,float DiameterCircle)
 	{
 		RadiusCircle = 0.0f;
 		DiameterCircle = 0.0f;
 	}
 	
-	void InPutDataCircle()
+	void InputDataCircle()
 	{
 		float EntryRadius;
 		float EntryDiameter;
 		std::cout << "what is the area of Your Circle:" << std::endl;
-		std::cin >>EntryRadius;
-		while (!std::cin.good() || EntryRadius < 0)
+		std::cin >> EntryRadius;
+		while (!std::cin.good() || EntryRadius < 0.0f)
```

Comment:
Why are you asking for a radius if you never use it?

Reply:
![image](https://user-images.githubusercontent.com/36427649/180801287-8ff9e022-b99e-4d1e-93e0-8c1fec48dc79.png)
 I use the radius here for find the area